### ISSUE TITLE
`caddy trust` using wrong path?

### ISSUE THREAD

**Issue by AvverbioPronome** [caddyserver/caddy#4759]
````
root@Quake:/etc/caddy# caddy trust
trust: requesting CA info: caddy responded with error: HTTP 404: {"error":"resource not found: /pki/local/certificates"}

root@Quake:/etc/caddy# curl -i http://localhost:2019/pki/ca/local/certificates
HTTP/1.1 200 OK
Content-Type: application/pem-certificate-chain
Date: Tue, 03 May 2022 10:31:23 GMT
Content-Length: 1307

-----BEGIN CERTIFICATE-----
MIIBxzCCAW2gAwIBAgIQUXrbdLvBVraOJK4IbejgEDAKBggqhkjOPQQDAjAwMS4w
...
f6AhQu6m1/94J8+34K48HRwKbaA3qoWa+wRi
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
MIIBpDCCAUqgAwIBAgIRAMRVzeLw5cO2jxcSUYYlHFswCgYIKoZIzj0EAwIwMDEu
...
faTTY4QCIQDjH7KSkJ/R3OpKW9GBOdF8bkL5rczAwa3iP2yx05kEtw==
-----END CERTIFICATE-----
root@Quake:/etc/caddy# caddy version
v2.5.0 h1:eRHzZ4l3X6Ag3kUt8nj5IxATprhqKq/wToP7OHlXWA0=
````

On Ubuntu 22.04, installed via apt and run via systemd.

**Comment by mohammed90** [comment]
This was fixed in #4730. Please build from `master` if you need an immediate fix.

**Comment by AvverbioPronome** [comment]
> This was fixed in #4730. Please build from `master` if you need an immediate fix.

Nah, I'll wait for the apt release.

Thank you.
